src/mysql-util/src/decoding.rs

```diff
@@ -151,13 +151,14 @@ fn pack_val_as_datum(
                             Value::Int(val) => {
                                 // Enum types are provided as 1-indexed integers in the replication
                                 // stream, so we need to find the string value from the enum meta
-                                let enum_val = e.values.get(usize::try_from(val)? - 1).ok_or(
-                                    anyhow::anyhow!(
-                                        "received invalid enum value: {} for column {}",
-                                        val,
-                                        col_desc.name
-                                    ),
-                                )?;
+                                let enum_val =
+                                    e.values.get(usize::try_from(val)? - 1).ok_or_else(|| {
+                                        anyhow::anyhow!(
```

Comment:
nit: are we still wanting to remove anyhow?

Reply:
I think yes, but not urgently, and not part of this PR.